freight/models/app.py

```diff
@@ -82,3 +82,36 @@ def get_default_ref(self, env):
         if not data:
             return DEFAULT_REF
         return data.get('default_ref', DEFAULT_REF)
+
+    def get_current_sha(self, env):
```

Comment:
I'm not a huge fan of overloaded models. I've done it in the past and regretted it as it puts too much code in one place. What was the goal with moving them here?